builtin/logical/pki/path_root.go

```diff
@@ -383,32 +383,30 @@ func (b *backend) pathIssuerSignIntermediate(ctx context.Context, req *logical.R
 		resp.AddWarning("This mount hasn't configured any authority access information fields; this may make it harder for systems to find missing certificates in the chain or to validate revocation status of certificates. Consider updating /config/urls with this information.")
 	}
 
+	caChain := append([]string{cb.Certificate}, cb.CAChain...)
+
 	switch format {
 	case "pem":
 		resp.Data["certificate"] = cb.Certificate
 		resp.Data["issuing_ca"] = signingCB.Certificate
-		if cb.CAChain != nil && len(cb.CAChain) > 0 {
-			resp.Data["ca_chain"] = cb.CAChain
-		}
+		resp.Data["ca_chain"] = caChain
 
 	case "pem_bundle":
 		resp.Data["certificate"] = cb.ToPEMBundle()
```

Comment:
This is what makes me think this is a good idea: this gives you leaf + chain of the cert in the `certificate` field. :-) Avoids the need for concat.